feat(respect): introduce global execution timeout-timer (#2003)
Redocly/redocly-cli: .changeset/silent-suits-kiss.md

---
"@redocly/respect-core": minor
"@redocly/cli": minor
---

> Add global execution timeout timer to `respect` command execution in order to prevent infinite test runs. The timer can be configured using the `RESPECT_TIMEOUT` environment variable (defaults to 1 hour).

Reviewer: ```suggestion
Added global execution timeout timer to `respect` command execution to prevent infinite test runs. You can configure this timer using the `RESPECT_TIMEOUT` environment variable (defaults to 1 hour).
```
